runtime/v2/runc/service.go

```diff
@@ -660,13 +723,13 @@ func (s *service) checkProcesses(e runcC.Exit) {
 				}
 			}
 			p.SetExited(e.Status)
-			s.events <- &eventstypes.TaskExit{
+			s.sendL(&eventstypes.TaskExit{
```

Comment:
`send` here right?

Reply:
send for this because we only want to send exit events if the start lock is not held